taskt/Core/Automation/Commands/Error Handling Commands/TryCommand.cs

```diff
@@ -1,88 +1,87 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Windows.Forms;
-using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
 using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Error Handling Commands")]
-    [Attributes.ClassAttributes.Description("This command allows embedding commands and will automatically move to the 'catch' handler")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to handle potential errors that could occur.")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Error Handling Commands")]
+    [Description("This command allows embedding commands and will automatically move to the 'catch' handler")]
```

Comment:
Please use punctuation in Description as per guideline.

Reply:
Done.